### DISCUSSION TITLE
求助，需要将阿里云资源将整个文件夹推送到aria2下载。需要怎么设置才行

### DISCUSSION THREAD

**Discussion by raosongyu** [AlistGo/alist#3992] [Q&A]
因资源已经分好类，带字幕，如果直接下载到默认文件夹，感觉太乱了，想将整个文件夹推送过去。目前显示文件夹推送成功，但ariaNG没有下载内容

**Comment by Rektyfikowany** [comment]
因文件夹没有link暂时不能直接推送，可以考虑挂载一个本地存储做存储间复制，或者使用webdav拷贝工具

**Comment by raosongyu** [comment]
感谢感谢
